src/common/common_funcs.h

```diff
@@ -126,4 +126,26 @@ constexpr std::size_t Size(const C& c) {
     }
 }
 
+namespace Literals {
+constexpr u64 operator""_KiB(unsigned long long int x) {
+    return 1024ull * x;
```

Comment:
We use uppercase suffixing to clearly differentiate between 1 and l in fonts and prevent misreads.

```suggestion
    return 1024ULL * x;
```